### DISCUSSION TITLE
Welcome to nextgen-dmac Discussions!

### DISCUSSION THREAD

**Discussion by jonmjoyce** [ioos/nextgen-dmac#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamesdoyle21** [comment]
Here is a great resource to get caught up on the background of this project:

[Topic2_SteeringCommittee_Kickoff.pdf](https://github.com/asascience-open/nextgen-dmac/files/9397516/Topic2_SteeringCommittee_Kickoff.pdf)

**Comment by jamesdoyle21** [comment]
If you are new to the GitHub community, check out these links [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) and [here](https://www.youtube.com/watch?v=gvvvdearAPE) to get started!
